components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java

```diff
@@ -35,6 +35,9 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * IdentityGovernanceServiceImpl.
```

Comment:
Shall we make this meaningful?

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767